Formula/xxhash.rb

```diff
@@ -22,6 +25,7 @@ class Xxhash < Formula
   def install
     system "make"
     system "make", "install", "PREFIX=#{prefix}"
+    prefix.install "cli/COPYING"
```

Comment:
Previously, both licenses were included in `LICENSE` file. They were split up in current release to `LICENSE` (2-clause BSD) and `cli/COPYING` (GPLv2). As license files have different name, I think we can just install the other license into same prefix.